spec/manifest_spec.rb

```diff
@@ -226,6 +226,22 @@
       ])
     end
 
+    context 'when products is a string that is not a location' do
+      before { manifest_hash[:location] = 'cool_location' }
+
+      it 'defaults to support without raising an error' do
```

Comment:
Should this be 'defaults to support without raising an error'?

Reply:
Nope, thanks